client/landing/jetpack-cloud/sections/scan/main.tsx

```diff
@@ -152,6 +154,22 @@ class ScanPage extends Component< Props > {
 		);
 	}
 
+	handleOnScanNow = () => {
+		const {
+			dispatchRecordTracksEvent,
+			site,
+			dispatchRequestScanEnqueue,
+			dispatchStartScanOptimistically,
+		} = this.props;
+
+		dispatchRecordTracksEvent( 'calypso_scan_run', {
```

Comment:
Are we still using `calypso` prefixes for Cloud events?

Reply:
Yes, but I think nothing has been defined yet. I wrote a bit about that in `pbtFFM-ci-p2`. For context, I tried to use `jetpack_cloud` before but I couldn't because we need to first whitelist that prefix (and maybe do a couple of other things).

Reply:
Good question. I remember this topic was discussed, but I don't remember the outcome, if there was one. @rcanepa's post here is the only thing I've been able to find so far: pbtFFM-ci-p2